assets/app/scripts/controllers/settings.js

```diff
@@ -68,24 +68,29 @@ angular.module('openshiftConsole')
     DataService.list("limitranges", $scope, function(limitRanges) {
```

Comment:
It seemed like an unnecessary websocket to be opening.  Your limit ranges on your project will almost never change. They can only be modified by cluster admins.